mw-webapp/src/utils/useAuth.ts

```diff
@@ -0,0 +1,26 @@
+import {GoogleAuthProvider, getAuth, signInWithRedirect, getRedirectResult} from "firebase/auth";
+import {secondaryApp} from "src/firebase";
+
+export const useAuth = () => {
+  const provider = new GoogleAuthProvider();
+  const auth = getAuth(secondaryApp);
+
+  const handleGoogleSignIn = async () => {
+    try {
+      await signInWithRedirect(auth, provider);
+      const result = await getRedirectResult(auth);
+      if (result) {
+        const user = result.user;
+        console.log(user);
```

Comment:
Console.log

Reply:
I put console.log here to prevent ESLint error - unused variable user.